server/controllers/profile.controller.js

```diff
@@ -131,6 +131,77 @@ class ProfileController {
     };
     return Response(res, 200, 'Successfully unfollowed user', user);
   }
+
+  /**
+   * get followers
+   * @param {object} req
+   * @param {object} res
+   * @returns {object} response
+   */
+
+  static async getFollowers(req, res) {
+    const {
+      user: { id: userId }
+    } = req;
+
+    const user = await User.findByPk(userId);
+
+    const followers = await user.getFollowers({
+      attributes: ['id', 'email', 'firstname', 'lastname'],
+    });
+
+    if (followers.length === 0) {
+      return Response(res, 200, 'User currently does not have followers');
+    }
+
+    const follower = {
+      count: followers.length,
+      followee: user.username,
+      followers: followers.map(follow => ({
+        id: follow.id,
+        firstname: follow.firstname,
+        lastname: follow.lastname,
+        email: follow.email
+      })),
+    };
+
+    return Response(res, 200, 'Followers returned successfully', follower);
+  }
+
+  /**
+   * get followee
+   * @param {object} req
+   * @param {object} res
+   * @returns {object} response
+   */
+
+  static async getFollowee(req, res) {
+    const {
+      id
+    } = req.user;
+
+    const user = await User.findByPk(id);
+    const followee = await user.getFollowings({
+      attributes: ['id', 'email', 'firstname', 'lastname']
+    });
+
+    if (followee.length === 0) {
+      return Response(res, 200, 'User is not a follower');
```

Comment:
@samorahno can we change this to `User does not follow anyone currently` or something along that line?